libs/server/Storage/Session/ObjectStore/ListOps.cs

```diff
@@ -84,6 +85,7 @@ public unsafe GarnetStatus ListPush<TObjectContext>(ArgSlice key, ArgSlice eleme
             var status = RMWObjectStoreOperation(key.ToArray(), element, out var output, ref objectStoreContext);
             itemsDoneCount = output.countDone;
 
+            itemBroker.HandleCollectionUpdate(key.Span.ToArray());
```

Comment:
ideally the broker deals with spans directly so we are not allocting unless it is absolutely necessary.